specificatie/ouder.yaml

```diff
@@ -6,6 +6,10 @@ info:
 paths: {}
 components:
   schemas:
+    OuderAanduiding:
+      type: string
+      pattern: ^[1|2]$
```

Comment:
wanneer we hier geen code+omschrijving leveren, zou ik de waarden "ouder1" of "ouder2" gebruiken.

Reply:
dat scheelt ook weer een heleboel aanpassingen in features

Reply:
Ik heb deze waarden gebaseerd op de uitkomst van de discussie bij #941 .  Heb ik die uitkomst nu verkeerd geïnterpreteerd (Voorstel van Melvin letterlijk geïmplementeerd) of stel jij die uitkomst ter discussie ? 
Overigens zijn al die aanpassingen in features in deze PR gedaan volgens mij.

Reply:
oh ja, dat was om de code van ouderAanduiding te laten aansluiten bij de code voor gezag minderjarige.